### PR TITLE
Listing details

### PR DESCRIPTION
**Pull request by archmagos**
Rudimentary individual listing pages added

### CODE REVIEW

File: _Express/routes/listings.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var express = require('express');
 var router = express.Router();
 
-var api = require('../public/javascripts/jsonRequest.js')
+var api = require('../public/javascripts/jsonRequest.js');
```
Comment (JohnForster) [inline _Express/routes/listings.js:4]: Pedant.

File: _Express/views/listing.jade [hunk 2]
```diff
@@ -0,0 +1,10 @@
+extends layout
+
+block content 
+ div.listing_display.
+   <img src=#{listingsJSON.listings[id-1].image_url} style="width:300px"><br>
+   Title: #{listingsJSON.listings[id-1].title}<br>
+   Desc: #{listingsJSON.listings[id-1].description}<br>
+   Capacity: #{listingsJSON.listings[id-1].capacity}<br>
+   Price: #{listingsJSON.listings[id-1].price} gold pieces<br>
+   Rating: #{'💀'.repeat(listingsJSON.listings[id-1].rating)}<br><br>
```
Comment (JohnForster) [inline _Express/views/listing.jade]: To keep this DRY, could specify `listingJSON: data.listings[id-1]` on line 9 of `/routes/listing.js` and use `listingJSON` in the view?

**Review by JohnForster — CHANGES_REQUESTED** [review]

File: _Express/routes/listings.js (hunk 1 above)
Comment (archmagos) [reply _Express/routes/listings.js:4]: :^)

**Comment by archmagos** [comment]
Changes made, please review 💯

**Review by JohnForster — APPROVED** [review]

**Pull request merged**